src/pages/Admin/BackgroundSelect/BackgroundSelect.scss

```diff
@@ -0,0 +1,35 @@
+@import "scss/constants";
+
+.BackgroundSelect {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: $spacing--xl 0;
+  position: relative;
+  background-size: cover;
+  background-color: $dark;
+  background-position: center;
+
+  &__map-grid {
+    display: grid;
+    width: 50%;
+    grid-gap: 1rem;
```

Comment:
variable

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/e3b301bfa611c387d3e2ac24b7195994415ffc70